docs/docsite/rst/dev_guide/developing_collections_path.rst

```diff
@@ -0,0 +1,160 @@
+.. _developing_collections_path:
+
+*******************************
+Ansible collection creator path
+*******************************
+
+The guide's purpose and overview
+================================
+
+.. note::
+
+  If you are totally unfamiliar with the collection technology, first take a look at the :ref:`Using Ansible collections guide<collections>`.
+
+Ansible collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins.
+A typical collection addresses a set of related use cases. For example, the ``community.dns`` collection includes modules and plugins to work with DNS.
+
+You can install collections made by others or share yours with the community through a distribution server such as Ansible Galaxy. Certified collections can be published to the Red Hat Automation Hub, a part of the Red Hat Ansible Automation Platform.
```

Comment:
```suggestion
You can install collections made by others or share yours with the community through a distribution server such as `Ansible Galaxy <https://galaxy.ansible.com/ui/>`_. Certified collections can be published to the Red Hat Automation Hub, a part of the Red Hat Ansible Automation Platform.
```

Reply:
@oraNod - what do you think about including the breadcrumb above that mentions AAP?